contracts/pm/LivepeerETHTicketBroker.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^0.4.25;
+// solium-disable-next-line
+pragma experimental ABIEncoderV2;
+
+import "../ManagerProxyTarget.sol";
+import "../token/IMinter.sol";
+import "../bonding/IBondingManager.sol";
+import "../rounds/IRoundsManager.sol";
+import "./TicketBroker.sol";
+
+
+contract LivepeerETHTicketBroker is ManagerProxyTarget, TicketBroker {
+    constructor(
+        address _controller,
+        uint256 _minPenaltyEscrow,
+        uint256 _unlockPeriod
+    ) 
+        Manager(_controller)
+        // TODO: Consider using a initializer instead of an
+        // explicit constructor in base TicketBroker since
+        // upgradeable proxies do not use explicit constructors
+        TicketBroker(_minPenaltyEscrow, _unlockPeriod)
+        public
+    {}
+
+    function setMinPenaltyEscrow(uint256 _minPenaltyEscrow) external onlyControllerOwner {
+        minPenaltyEscrow = _minPenaltyEscrow;
+    }
+
+    function setUnlockPeriod(uint256 _unlockPeriod) external onlyControllerOwner {
+        unlockPeriod = _unlockPeriod;
+    }
+
+    function fundDeposit()
+        external
+        payable
+        processDeposit(msg.sender, msg.value)
+    {
+        minter().trustedDepositETH.value(msg.value)();
+    }
+
+    function fundPenaltyEscrow()
+        external
+        payable
+        processPenaltyEscrow(msg.sender, msg.value)
+    {
+        minter().trustedDepositETH.value(msg.value)();
+    }
+
+    function withdrawTransfer(address _sender, uint256 _amount) internal {
+        minter().trustedWithdrawETH(_sender, _amount);
+    }
+
+    function winningTicketTransfer(address _recipient, uint256 _amount) internal {
+        // TODO: Consider changing this to the ticket creation round
```

Comment:
Just noting that our latest decision here is to indeed change this to the ticket creation round, as we are planning to use rounds as the mechanism for ticket expiration as well.

@yondonfu makes sense?

Reply:
Yep I think we can remove the todo in the PR that implements the updated ticket expiration logic